feature/onboarding/src/main/java/com/teamsolply/solply/onboarding/component/ProgressBar.kt

```
@@ -0,0 +1,64 @@
+                .fillMaxWidth(size)
+                .height(8.dp)
+                .background(SolplyTheme.colors.gray900)
+                .animateContentSize()
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun ProgressBarPreview() {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)